### ISSUE TITLE
[Masthead] Overflow behavior - Doing tab key from the 4th menu item focus reaches to the 5th and 6th menu item but its is not visually available for the users

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5894]
## Detailed description

Issue:
Doing tab key from the 4th menu item focus reaches to the 5th and 6th menu item but its is not visually available for the users.

Expected:
Menu item should be visible for the users when keyboard focus reaches to the 5th or 6th menu item.

Note:
This defect does not exist on react staging.

## Additional information
-

**Comment by RobertaJHahn** [comment]
Ari has decided to do this dev work.  Removed the HC ownership label, rmoved the consulting description and increased the story points from 1 to 3.

**Comment by proeung** [comment]
@RobertaJHahn Sounds good! Thanks for letting us know.
